search.ipynb

```diff
@@ -1044,6 +770,87 @@
     "* `search(self, problem)`: This method is used to search a sequence of `actions` to solve a `problem`."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us now define a Simple Problem Solving Agent Program. We will create a simpleAgent class which will inherit from the abstract class `SimpleProblemSolvingAgentProgram` and overides it methods. We will create a simple intelligent vacuum agent which can be in any one of the following states. It will move to any other state depending upon the current state as shown in the picture by arrows:\n",
+    "<img src=\"files/images/simple_probem_solving_agent.jpg\">"
```

Comment:
Use Markdown to show the image instead of the html tag.

`![simple problem solving agent](images/simple_problem_solving_agent.jpg)` should do the trick. (Don't forget to fix the image name!)

Reply:
I changed it but wanted to know why is it better to use markdown and not html tag? because I say in some other file the html tag? @MrDupin